integration-test/src/main/java/com/sequenceiq/it/cloudbreak/context/TestContext.java

```diff
@@ -263,7 +263,14 @@ public InstanceAwait getInstanceAwait() {
         Log.whenException(LOGGER, action.getClass().getSimpleName() + " action on " + entity + " by " + who);
 
         try {
-            return doAction(entity, clientClass, action, who.getAccessKey());
+            String message = String.format("Expected exception with message (%s) has not been thrown at action (%s)!",
```

Comment:
What will happen if the `runningParameter.getExpectedMessage()` is empty or null?

Reply:
This is going to show an `()` in the message. I think no other issue is going to be generated.